desc/compute/_equil.py

```diff
@@ -462,6 +462,74 @@ def _W_B(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="W_B(r)",
+    label="W_B(\\rho)",
+    units="J",
+    units_long="Joules",
+    description="Plasma magnetic energy enclosed by flux surfaces",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["|B|", "e_theta", "e_zeta", "Z"],
+)
+def _W_B_of_r(params, transforms, profiles, data, **kwargs):
```

Comment:
not sure I totally understand this. It looks like divergence theorem or something to turn the volume integral into a surface integral but what is the divergence of?

Reply:
I took the divergence theorem that we use to compute `V(r)` ([see code here](https://github.com/PlasmaControl/DESC/blob/master/desc/compute/_geometry.py#L39)) and just multiplied it by B^2. 

I wasn't sure about the math, but this matches the VMEC output for `beta_vol` to within a few percent.